storybook/src/foundation/design-tokens/links.docs.mdx

```diff
@@ -0,0 +1,56 @@
+{/* @license CC0-1.0 */}
+
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Brand/Design Tokens/Links" />
+
+# Links
+
+We have defined a group of link styles for all link components.
```

Comment:
This mentions the reason (multiple link components) before the solution (common tokens), emphasizes the relation they codify, and works around using ‘we’.

```suggestion
Several link components share the same styles.
This set of common tokens represents that relation.
```